service-front/module/Application/view/application/pages/abandoned_flow.twig

```diff
@@ -0,0 +1,75 @@
+{% extends "layout/layout" %}
+{% block content %}
+    {% include 'layout/id_check_banner.twig' with details_data %}
+    <form method="post" class="govuk-!-width-two-thirds">
+        <div class="govuk-width-container govuk-!-padding-top-5">
+            <h1 class="govuk-heading-xl">
+                {% block title %}
+                    Exit identity check?
+                {% endblock %}
+            </h1>
+            <div class="govuk-form-group">
+                <input type="hidden"  name="last_page" value="{{ last_page }}">
```

Comment:
Is `last_page` being used for anything in this form?

Reply:
It'll be used in the progess tracking feature. This branch was initially going to include the progress tracking, but I decided to split the tickets.